rslearn/data_sources/planetary_computer.py

```diff
@@ -151,20 +167,40 @@ def from_config(config: RasterLayerConfig, ds_path: UPath) -> "PlanetaryComputer
 
     def _load_client(
         self,
-    ) -> tuple[pystac_client.Client, pystac_client.CollectionClient]:
+    ) -> pystac_client.Client:
```

Comment:
This interface change is the breaking part that will require us/me to modify the `Monthly*` datasources in rslp.